easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart

```diff
@@ -103,13 +119,18 @@ class NftGridViewItem extends StatelessWidget {
                   Container(
                     decoration: BoxDecoration(
                       borderRadius: BorderRadius.circular(1.h),
-                      color: context.read<CreatorHubViewModel>().selectedCollectionType == CollectionType.draft ? EaselAppTheme.kLightRed : EaselAppTheme.kDarkGreen,
+                      color: context.read<CreatorHubViewModel>().selectedCollectionType == CollectionType.draft
+                          ? EaselAppTheme.kLightRed
+                          : EaselAppTheme.kDarkGreen,
                     ),
                     padding: EdgeInsets.symmetric(horizontal: 6.w, vertical: 3.h),
                     margin: EdgeInsets.symmetric(horizontal: 6.w, vertical: 6.h),
                     child: Text(
-                      context.read<CreatorHubViewModel>().selectedCollectionType == CollectionType.draft ? "draft".tr() : "published".tr(),
-                      style: EaselAppTheme.titleStyle.copyWith(color: EaselAppTheme.kWhite, fontSize: isTablet ? 8.sp : 11.sp),
+                      context.read<CreatorHubViewModel>().selectedCollectionType == CollectionType.draft
+                          ? "draft".tr()
```

Comment:
don't use `"draft".tr()` directly instead use `LocaleKeys.draft.tr()` which is in auto generated file by easy_localization package. 
use this command `flutter pub run easy_localization:generate -f keys -o locale_keys.g.dart --source-dir ./i18n/` to generate that file